inference-engine/src/transformations/include/transformations/depth_to_space_fusion.hpp

```diff
@@ -12,23 +12,28 @@
 #include <ngraph/pass/graph_rewrite.hpp>
 #include "transformations/utils/pass_param.hpp"
 
+/**
+ * @brief ngraph namespace
```

Comment:
namespaces should be documented only once per project. 

BTW, documenting namespaces itself is needed because doxygen will not parse classes / functions / etc from namespace otherwise.

Reply:
done